### ISSUE TITLE
Support for more than 32726 iteration

### ISSUE THREAD

**Issue by pyscripter** [pyscripter/XLRisk#20]
Currently if you specify as number of iteration a number > 32726 it is not accepted and does not work.  This limitation should be removed.

**Comment by maurofughi** [comment]
Hi, I saw you closed this so I figured you had removed that limit, so I downloaded the new version of the add-in and installed in my excel and tested it, and I noticed the following issues:
1) when entering the new number of iterations in the ribbon field directly is OK. But if I click on SETUP and enter the number of iterations in the setup popup window I get an error (see pic1 below). Then if I click on debug I see the second picture below).
If I just close the debug and run XLRisk starts running.
2) When XLRisk runs now the STOP button is now greyed out so not possible to stop a simulation (have to close excel using windows task manager).
Then when the run ends data seem generated correctly and I can see results.
![image](https://github.com/user-attachments/assets/f021f5fb-763b-4bea-a834-be8730529a10)
![image](https://github.com/user-attachments/assets/917168e3-c7f7-4d6c-9d85-b673a2bd84b8)

**Comment by maurofughi** [comment]
SO far I have tried up to 100,000 iterations as with my model takes 2 hours, will try with a larger number overnight.

**Comment by pyscripter** [comment]
> 1. if I click on SETUP and enter the number of iterations in the setup popup window I get an error

Fixed

> 2. When XLRisk runs now the STOP button is now greyed out

Also fixed.  

Please test and report back.

Also note that the number of iterations is limited by the rows in the worksheet (about 1,000,000)

**Comment by maurofughi** [comment]
Yep, just had one run and no error in setting window, and the stop button is available. Looks all good, thanks!
